jck/jck.mk

```diff
@@ -107,15 +107,37 @@ endif
 
 JCK_CMD_TEMPLATE = $(JAVA_TO_TEST) -Djvm.options=$(Q)$(JVM_OPTIONS)$(Q) -Dother.opts=$(Q)$(OTHER_OPTS)$(Q) -cp $(TEST_ROOT)/jck/jtrunner/bin JavaTestRunner resultsRoot=$(REPORTDIR) testRoot=$(TEST_ROOT) jckRoot=$(JCK_ROOT) jckversion=$(JCK_VERSION) configAltPath=$(CONFIG_ALT_PATH) $(APPLICATION_OPTIONS)
 
+USE_MULTINODE = false
+OS:=$(shell uname -s | tr "[:upper:]" "[:lower:]")
+ARCH:=$(shell uname -m | tr "[:upper:]" "[:lower:]")
+ifneq ($(filter aix os/390, $(OS)),)
+   USE_MULTINODE = true
+endif
+ifeq ($(OS),linux)
+   ifeq ($(ARCH),ppc64le)
+      USE_MULTINODE = true
+   else ifeq ($(ARCH),s390x)
+      USE_MULTINODE = true
+   endif
+endif
```

Comment:
We should be able to use `$(SPEC)`. See https://github.com/adoptium/aqa-tests/blob/master/openjdk/openjdk.mk#L110 for example.

Reply:
That's simpler! Updated. Tested with AIX: Grinder_JCK/1125, xLinux: Grinder_JCK/1126.